bundles/org.openhab.binding.xmppclient/pom.xml

```
@@ -15,13 +15,14 @@
   <name>openHAB Add-ons :: Bundles :: XMPPClient Binding</name>
 
   <properties>
-    <smack.version>4.3.3</smack.version>
+    <smack.version>4.5.0-beta2</smack.version>
```

There is a newer beta:

```suggestion
    <smack.version>4.5.0-beta5</smack.version>
```

Java 8 support was dropped so you should use `smack-java11` now.

See: https://discourse.igniterealtime.org/t/smack-4-5-0-beta5-released/94796